extensions/xcode/src/components/xcode-simulator-list/xcode-simulator-list-dropdown.component.tsx

```diff
@@ -0,0 +1,21 @@
+import { List } from "@raycast/api";
+import { XcodeSimulatorStateFilter } from "../../models/xcode-simulator/xcode-simulator-state-filter.model";
+
+export function XcodeSimulatorStateDropdown(props: {
+  onSimulatorStateFitlterChange: (newValue: XcodeSimulatorStateFilter) => void;
+}) {
+  const { onSimulatorStateFitlterChange } = props;
+  return (
+    <List.Dropdown
+      tooltip="Select a simulator state"
+      storeValue={true}
```

Comment:
I'm not 100% sure about saving the selected filter state 🤔
Maybe this could cause an unexpected behaviour when a user reopens the command.

Reply:
I prefer it to be stored since when I do use the manage simulators command I use it multiple times in a row and feel like it's convenient to keep the same filtering as a preference.
Since dropdowns are a core feature of Raycast present in many extensions, I don't see how it could lead to unexpected behavior it just acts as a preference while you still are able to change easily it if needed.
Adding to that, I think that people don't use this command for the same purpose, some of us might use this command to boot up multiple simulators so they might want to keep a `Shut down` filter, and some are using it to open links or access to local files so they only need to see `Booted` ones by default.

Reply:
Alright! Good point 👍